code/modules/urist/items/uristmed.dm

```diff
@@ -39,3 +39,174 @@ Space for all Urist-done, non-pill medical items. Please keep it tidy, as usual.
+		return 1
+
+	if (istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		var/datum/organ/external/affecting = H.get_organ(user.zone_sel.selecting)
+
+		if(affecting.display_name == "head")
+			if(H.head && istype(H.head,/obj/item/clothing/head/helmet/space))
+				user << "<span class='warning'> You can't apply [src] through [H.head]!</span>"
+				return 1
+		else
+			if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/space))
+				user << "<span class='warning'> You can't apply [src] through [H.wear_suit]!</span>"
+				return 1
+
+		if(affecting.status & ORGAN_ROBOT)
+			user << "<span class='warning'> This isn't useful at all on a robotic limb..</span>"
+			return 1
+
+		H.UpdateDamageIcon()
+
+		if(affecting.open == 0)
+			if(!affecting.bandage())
+				user << "<span class='warning'>The wounds on [M]'s [affecting.display_name] have already been bandaged.</span>"
+				return 1
+			else
+				for (var/datum/wound/W in affecting.wounds)
+					if (W.internal)
+						continue
+					if (W.current_stage <= W.max_bleeding_stage)
+						user.visible_message( 	"<span class='notice'> [user] presses [W.desc] on [M]'s [affecting.display_name] with the [src].</span>", \
+										"<span class='notice'> You press [W.desc] on [M]'s [affecting.display_name] with the rag, trying to stop the bleeding.</span>" )
+						src.bloodvolume += 1
+						src.checkvolume()
+						if(prob(33))
+							W.germ_level += 1 //rags are rarely sanitary
+						spawn(10)
+							if(prob(20))
+								affecting.status &= ~ORGAN_BLEEDING
+								user.visible_message( 	"<span class='notice'> [user] stops the bleeding of [W.desc] on [M]'s [affecting.display_name] with the [src]!</span>", \
+												"<span class='notice'> You manage to stop the bleeding of [W.desc] on [M]'s [affecting.display_name] with the [src]!</span>" )
+							else return 0
+
+
+					else
+						user <<	"<span class='warning'> This wound is not bleeding, use more advanced medical gear to heal the [W.desc] on [M]'s [affecting.display_name].</span>"
+						return 0
+		else
+			if (can_operate(H))        //Checks if mob is lying down on table for surgery
+				if (do_surgery(H,user,src))
+					return
+			else
+				user << "<span class='notice'>The [affecting.display_name] is cut open, you'll need more than a [src]!</span>"
+				return 0
+
+
+/obj/item/clothing/suit/verb/rip(mob/user as mob)
+	set name = "Rip Apart Suit"
+	set category = "Object"
+	set src in usr
+
+	var/mob/living/carbon/human/H = src.loc
+
+	if(!usr.canmove || usr.stat || usr.restrained())
+		return 0
+	if(!istype(H))
+		return 0
+	if(H.wear_suit == src)
+		user << "<span class='warning'>Deciding not to channel your inner barbarian, you think it's a good idea to take off your [src] first.</span>"
+		return
+
+	if(istype(src, /obj/item/clothing/suit/space) || istype(src, /obj/item/clothing/suit/armor) || istype(src, /obj/item/clothing/suit/wizrobe)) //add other protected instances as needed
```

Comment:
Or if you really want suit/storage to be rippable, you could always add a check for them to drop their contents before calling del() on the the suit.